data-analysis/src/regex.py

```diff
@@ -0,0 +1,2 @@
+alphabetic = r'[A-Za-zŻŹĆĄŚĘŁÓŃżźćńółęąś]'
```

Comment:
as soon as `re.UNICODE` flag is provided, `\w` should catch'em all

Reply:
My bad, that's true for python2. In version 3 `\w` should get them anyways (skipped that part of my suggestion where I think that blacklisting numbers and using \w would be safer). Conclusion -- nevermind